content-services/7.4/tutorial/sso/ldap.md

```diff
@@ -0,0 +1,296 @@
+
+3. Choosing a **Vendor** will auto-populate many of the fields.
+
+4. Enter the **Connection URL** for the LDAP instance in the format:
+    * `ldap//ldap.example.com:389` or
+    * `ldaps//ldap.example.com:636` for SSL-enabled installations
+
+5. Set the **Batch Size** and whether to use **Full Sync** and/or **Period Changed Users Sync** followed by the associated **Sync Periods**.
+
+6. Save the configuration.
+
+## Step 3: Configure Alfresco Content Service properties
+
+The properties listed that need to be set for Alfresco Content Services (ACS) are only those that are required for setting up SSO. They include the synchronization with an LDAP directory and the location of a SAML keystore. The Alfresco Share configuration file also requires updating to enable SSO.
+
+1. Use the following configuration parameters either in an `alfresco-global.properties` file, via the repository config map in Kubernetes or as environment variables in a docker-compose file:
+
+    | Property | Description |
+    | -------- | ----------- |
+    | authentication.chain | The authentication chain needs to be set for Keycloak and LDAP synchronization, for example `identity-service-1:identity-service,alfrescoNtlm-1:alfrescoNtlm,ldap-1:ldap`. |
+    | identity-service.auth-server-url | Keycloak's base URL, for example `https://keycloak.example.com/auth`. |
+    | identity-service.enable-basic-auth | Sets whether basic authentication is also supported by Keycloak, for example `true`. |
+    | identity-service.realm | The realm name configured in Keycloak for the Alfresco applications, for example `alfresco`. |
+    | identity-service.resource| The **Client ID** set up in Keycloak for Alfresco Content Services. The client needs to exist underneath the realm set for `identity-service.realm`, for example `alfresco`. |
+    | ldap.authentication.active | Sets whether LDAP authentication is enabled or not. This needs to be set to `false` to use SAML authentication via Keycloak, for example `false`. |
+    | ldap.synchronization.active | Sets whether LDAP synchronization is enabled or not. This needs to be set to `true` to sync users with the repository, for example `true`. |
+    | ldap.synchronization.java.naming. security.authentication | The mechanism to use to authenticate with the LDAP server, for example `simple`. |
+    | ldap.synchronization.java.naming. security.principal | The user principal name (UPN) of the account used to retrieve account details for all users and groups, for example `alfresco@domain.com`. |
+    | ldap.synchronization.java.naming.security.credentials | The password for the account set in `ldap.synchronization.java.naming.security.principal`, for example `secret`. |
+    | ldap.* | There are several optional [configuration]({% link content-services/latest/admin/auth-sync.md %}#ldapconfprops) and [synchronization]({% link content-services/latest/admin/auth-sync.md %}#synchronization-configuration-properties) properties. |
```

Comment:
Similar to the previous comment, make sure all links within `content-services/7.4/` remain in the same folder and not `content-services/latest/`. Check for other instances under `content-services/7.4/`.